include/dx_azure_iot.h

```diff
@@ -64,7 +62,8 @@ bool dx_isAzureConnected(void);
 /// <param name="messagePropertyCount"></param>
 /// <param name="messageContentProperties"></param>
 /// <returns></returns>
-bool dx_azurePublish(const void* message, size_t messageLength, DX_MESSAGE_PROPERTY** messageProperties, size_t messagePropertyCount, DX_MESSAGE_CONTENT_PROPERTIES* messageContentProperties);
+bool dx_azurePublish(const void *message, size_t messageLength, DX_MESSAGE_PROPERTY **messageProperties, size_t messagePropertyCount,
+                     DX_MESSAGE_CONTENT_PROPERTIES *messageContentProperties);
```

Comment:
It looks like this declaration got an extra LF inserted.

Reply:
ahhh I decided to run the formatter to bring up to the spec in .clang-format. It was a one of but I do appreciate it added complexity to the review. Sorry